docs/index.md

```diff
@@ -1,16 +1,31 @@
 Formula is a functional reactive framework built in Kotlin for managing state and side effects. It enables building 
 deterministic, composable, testable applications.
 
-### Quick example
-To demonstrate some of the concepts we will make a simple counter application. It shows the current count and 
+### Quick Start
+To demonstrate some of the concepts you will make a simple counter application. It shows the current count and 
 has two buttons to increment and decrement it.
 
 One of the best practises when working with Formula is to first think what the UI needs for rendering and what 
 actions the user will be able to take. This concept is called a `RenderModel` and is represented by a Kotlin data class.
 
+### Project Setup
+To get started create a new Android application called `Counter Application` with a blank activity. Next, rename (via refactor) 
```

Comment:
Maybe instead of this, I would recommend cloning the repository and going into `samples/counter`